src/runtime/particle-spec.ts

```diff
@@ -116,7 +116,7 @@ export class ConsumeSlotConnectionSpec {
 
   // Getters to 'fake' being a Handle.
   get isOptional(): boolean { return !this.isRequired; }
-  get direction(): string { return '`consume'; }
+  static get direction(): string { return '`consume'; }
```

Comment:
I think this doesn't want to be a static since it's attempting to implement the existing handle interface.  It just happens to return a constant in this implementation.

Reply:
Yeah fair. Will change in a bit.